Makefile.common_app

```diff
@@ -0,0 +1,95 @@
+#*******************************************************************************
+#   Ledger SDK
+#   (c) 2022 Ledger
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#*******************************************************************************
+
+COMMON_APP_FLAGS = 0x000
+
+#####################################################################
+#                            BLUETOOTH                              #
+#####################################################################
+ifeq ($(ENABLE_BLUETOOTH), 1)
+    COMMON_APP_FLAGS = 0x200  # APPLICATION_FLAG_BOLOS_SETTINGS
+    COMMON_APP_DEFINES += HAVE_BLE BLE_COMMAND_TIMEOUT_MS=2000 HAVE_BLE_APDU
+    COMMON_SDK_SOURCE_PATH += lib_blewbxx lib_blewbxx_impl
+endif
+
+#####################################################################
+#                               DEBUG                               #
+#####################################################################
+ifneq ($(DEBUG), 0)
+    COMMON_APP_DEFINES += HAVE_PRINTF
+    ifeq ($(TARGET_NAME),TARGET_NANOS)
+        COMMON_APP_DEFINES += PRINTF=screen_printf
+    else
+        COMMON_APP_DEFINES += PRINTF=mcu_usb_printf
+    endif
+else
+        COMMON_APP_DEFINES += PRINTF\(...\)=
+endif
+
+#####################################################################
+#                          COMMON DEFINES                           #
+#####################################################################
+ifeq ($(TARGET_NAME), TARGET_NANOS)
+    COMMON_APP_DEFINES += IO_SEPROXYHAL_BUFFER_SIZE_B=128
+else
+    COMMON_APP_DEFINES += IO_SEPROXYHAL_BUFFER_SIZE_B=300
```

Comment:
Most apps use this config 128 (LNS) and 300 (LNX / LNSP). But I found some that don't:

128 (LNS) and 128 (LNX / LNSP):
- app-sia
- app-u2f
- app-celo-validator-signer

72 (LNS) and 300 (LNX / LNSP):
- app-navcoin
- app-platon
- app-ethereum-ghsa-7hmg-5627-mv35
- app-ravencoin
- app-bitcoin-new
- app-theta
- app-ethereum.samsung
- app-xinfin
- app-ethereum

300 (LNS) and 300 (LNX / LNSP):
- app-decred
- app-woleet
- app-dgld
- app-nano
- app-passwords
- app-misc
- app-hodl
- app-bitcoin.samsung
- app-bitcoin
- app-icx

256 (LNS) and 300 (LNX / LNSP):
- app-radix

256 (LNS) and 256 (LNX / LNSP):
- app-zilliqa

Then I think someone should explain what's the impact of this buffer size and therefore which default value it should hold?

Reply:
If too custom we probably need an ENABLER for that? Or a DISABLER?